src/main/java/edu/tamu/app/service/manager/GitHubService.java

```diff
@@ -273,6 +274,11 @@ protected Member getMember(GHUser user) throws IOException {
             member = cachedMember.get();
         } else {
             String name = user.getName();
+
+            if (StringUtils.isEmpty(name)) {
+                name = user.getLogin();
```

Comment:
A ternary could be used to avoid the case of double assignment. 

```
String name = StringUtils.isEmpty(user.getName())
    ? user.getLogin()
    : user.getName();
```